app/views/curation_concerns/base/_form.html.erb

```diff
@@ -1,4 +1,5 @@
 <%= simple_form_for [main_app, @form], html: { multipart: true } do |f| %>
+    <div class="row">
```

Comment:
Even though it'll be a lot of changed lines, I'd like to see us have consistent indentation in this partial. Can we indent the inner content of this new row?